### PR TITLE
Remote catalog generation

### PR DESCRIPTION
**Pull request by pauloewerton**
this change integrates API, template and JSON input to generate a service catalog html.

### CODE REVIEW

File: src/main/java/cloud/fogbow/rcs/core/service/CatalogService.java [hunk 1]
```diff
@@ -60,7 +63,14 @@ public CatalogService() {
         }
         return services;
     }
-    
+
+    public String getServiceCatalog(String member, String service) throws FileNotFoundException {
```
Comment (pauloewerton) [inline src/main/java/cloud/fogbow/rcs/core/service/CatalogService.java]: this is a place holder. it's gonna be replaced by the proper integration with cache.

File: src/main/java/cloud/fogbow/rcs/api/http/request/Catalog.java [hunk 2]
```diff
@@ -5,6 +5,7 @@
 import org.apache.log4j.Logger;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
```
Comment (wassoares) [inline src/main/java/cloud/fogbow/rcs/api/http/request/Catalog.java]: Remove this unused import.

**Review by wassoares — APPROVED** [review]

File: src/main/java/cloud/fogbow/rcs/api/http/request/Catalog.java (hunk 2 above)
Comment (pauloewerton) [reply src/main/java/cloud/fogbow/rcs/api/http/request/Catalog.java]: done

File: src/main/java/cloud/fogbow/rcs/core/service/CatalogService.java [hunk 3]
```diff
@@ -60,7 +63,11 @@ public CatalogService() {
         }
         return services;
     }
-    
+
+    public String getServiceCatalog(String member, String service) {
+        return null;
```
Comment (pauloewerton) [inline src/main/java/cloud/fogbow/rcs/core/service/CatalogService.java:68]: this will be implemented when we integrate the remote services code.

**Review by ANKerD — APPROVED** [review]

**Review by raonismaneoto — APPROVED** [review]

**Pull request merged**